authzed/api/materialize/v0/watchpermissions.proto

```diff
@@ -66,6 +66,7 @@ message WatchPermissionsResponse {
 
 message PermissionChange {
   enum Permissionship {
+    // Unspecified is an unused default value that should not be encountered in normal operation.
```

Comment:
Perhaps I would reframe it as "it's used to signal some edge case triggered in the server side, and that something went wrong, and that clients are expected to just error out"